lib/screens/map_view.dart

```diff
@@ -61,11 +62,14 @@ class _MapViewScreenState extends State<MapViewScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-          title: Center(child: Text(widget.title)),
-          automaticallyImplyLeading: false),
+        centerTitle: true,
+        title: Image.asset('assets/images/logos_and_icons/logoshrink-removebg.png', height: 40),
+        automaticallyImplyLeading: false
+        ),
       body: Center(
           child: Stack(children: [
         map(context),
+        IgnorePointer(child: buildCompass())
```

Comment:
What does this widget do?

Reply:
It's what allows the beastie to still be clickable after adding the compass layer above it.